### ISSUE TITLE
Shuffle default config file so markup options are together and any other logical groups

### ISSUE THREAD

**Issue by jonbullock** [jbake-org/jbake#570]
See #487

**Comment by ancho** [comment]
Interesting. We could extend JbakeConfiguration to respect groups and add an option to the cli to print a list of available configuration options and current state, too. What do you think? Just a quick idea.

**Comment by jonbullock** [comment]
Yeah like that idea, another way for users to discover what options/features are available.

**Comment by alexmario74** [comment]
If this issue is still open I would like to try to fix it.
